src/c++/perf_analyzer/request_rate_manager.h

```diff
@@ -26,8 +26,8 @@
 #pragma once
```

Comment:
Update copyright to be correct in this file and elsewhere applicable.

Reply:
I only saw this file being an issue. It is fixed.